components/chart/index.tsx

```diff
@@ -55,24 +58,36 @@ function Chart({
     ZOOM_OUT: () => changeZoomLevel(10),
   };
 
+  const toggleDrawer = () => {
+    if (description !== '') {
+      setDescription('');
```

Comment:
I think no need for toggle action, it should be just close
```suggestion
      setSelectedModule(undefined);
```